packages/runtime/container-runtime/src/gc/gcEarlyAdoption.md

```diff
@@ -0,0 +1,130 @@
+# Garbage Collection: Advanced configuration for early adopters
+
+_For a technical overview of Garbage Collection, start with [GarbageCollection.md](./garbageCollection.md)_
+
+GC Sweep is not yet enabled by default, and until that time early adopters have several configuration options available
+for how to enable GC and monitor for any GC-impacting bugs that would need to be mitigated.
+
+Please refer to the section [Techniques Used for Configuration](./garbageCollection.md#techniques-used-for-configuration)
+before continuing, to ensure you're familiar with using "GC Options" and "Config Settings" for configuring GC.
+
+## What's on by default
+
+GC Mark Phase is enabled by default, which includes marking objects that are ready to be deleted as Tombstones.
+FF will log an informational event if/when a Tombstoned object is loaded - a scenario that would represent data loss if Sweep were enabled.
+The eventName for the Tombstone log ends with `GC_Tombstone_DataStore_Requested`.
+
+There's a similar event logged long before an object is Tombstoned. Ater only 7 days (configurable), an unreferenced object is considered
+"Inactive", and FF will log if an Inactive object is loaded as well.
+The eventName for the Inactive log ends with `InactiveObject_Loaded`.
+
+### Configuring InactiveObject timeout
```

Comment:
Why is this under `What's on by default`? Should't it be a separate topic?

Reply:
Since I just explained InactiveObject.  I can bump it up a level